tests/gis_tests/relatedapp/tests.py

```diff
@@ -100,9 +100,22 @@ def test_related_union_aggregate(self):
 
         # Ordering of points in the result of the union is not defined and
         # implementation-dependent (DB backend, GEOS version)
-        self.assertEqual({p.ewkt for p in ref_u1}, {p.ewkt for p in u1})
-        self.assertEqual({p.ewkt for p in ref_u2}, {p.ewkt for p in u2})
-        self.assertEqual({p.ewkt for p in ref_u1}, {p.ewkt for p in u3})
+        for point in [*u1, *u2, *u3]:
+            self.assertEqual(point.srid, 4326)
+
+        wkt_w = WKTWriter(trim=True, precision=6)
+        self.assertEqual(
+            {wkt_w.write(p).decode("utf-8") for p in ref_u1},
```

Comment:
I'm suspecting these test failures are due to version differences in GDAL/GEOS. 

There's [`equals_exact()`](https://docs.djangoproject.com/en/5.0/ref/contrib/gis/geos/#django.contrib.gis.geos.GEOSGeometry.equals_exact) which accepts a tolerance. That would be useful here, but what's tricky here is that we're comparing two sets. I'm not sure the Points can be ordered in a deterministic(?) way first.